pkg/ebpf/tracer_windows.go

```diff
@@ -45,7 +171,46 @@ func (t *Tracer) getConnections(active []ConnectionStats) ([]ConnectionStats, ui
 
 // GetStats returns a map of statistics about the current tracer's internal state
 func (t *Tracer) GetStats() (map[string]interface{}, error) {
-	return nil, ErrNotImplemented
+	if t.driverHandle == syscall.InvalidHandle {
+		return nil, fmt.Errorf("Problem with handle cannot get stats.")
+	}
+
+	var (
+		bytesReturned uint32
+		stats         C.struct_driver_stats
+		statbuf       = make([]byte, C.sizeof_struct_driver_stats)
+		ioctlcd       = ctl_code(NETWORK_DEVICE_TYPE_CTL_CODE, DDFILTER_IOCTL_GETSTATS, uint32(0), uint32(0))
+	)
+
+	err := syscall.DeviceIoControl(t.driverHandle, ioctlcd, nil, 0, &statbuf[0], uint32(len(statbuf)), &bytesReturned, nil)
```

Comment:
If we can get the stats object from other calls that will occur in the pulling of packets, it might be worth updating the variables through that method rather than making a  separate call here.  

Currently, the expvarStats() function means that we will be hitting the driver every 5 seconds with a call. I don't know if that means we might want to open up another, specific handle on the driver for pulling stats.